mousestyles/path_diversity/tests/test_smooth_noise.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+
+import pandas as pd
+
+from mousestyles.path_diversity import smooth_noise
+from mousestyles.path_diversity import path_index
+
+
+def test_smooth_noise():
+    movement = {'t': pd.Series([0., 0.02, 0.04, 0.06], index=[0, 1, 2, 3]),
+                'x': pd.Series([0., 0., 0.1, 0.2], index=[0, 1, 2, 3]),
+                'y': pd.Series([0., 1., 0., 1.], index=[0, 1, 2, 3]),
+                'isHB': pd.Series(['No', 'No', 'No', 'No'],
+                                  index=[0, 1, 2, 3])}
+    movement = pd.DataFrame(movement)
+    paths = path_index(movement, 1, 1)
```

Comment:
Correct me if I'm wrong, but since path_index is part of the path_diversity init file, it is not necessary. It would have returned an error otherwise, right?